static/app/components/events/eventViewHierarchy.tsx

```diff
@@ -0,0 +1,95 @@
+import {useState} from 'react';
+import isEqual from 'lodash/isEqual';
+
+import {getAttachmentUrl} from 'sentry/components/events/attachmentViewers/utils';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {tn} from 'sentry/locale';
+import {EventAttachment} from 'sentry/types';
+import {uniqueId} from 'sentry/utils/guid';
+import {useQuery} from 'sentry/utils/queryClient';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+import EventDataSection from './eventDataSection';
+import {ViewHierarchy, ViewHierarchyData} from './viewHierarchy';
+
+const DEFAULT_RESPONSE: ViewHierarchyData = {rendering_system: '', windows: []};
+const FIVE_SECONDS_IN_MS = 5 * 1000;
+
+function fillWithUniqueIds(hierarchy) {
+  return {
+    ...hierarchy,
+    id: uniqueId(),
+    children: hierarchy.children?.map(fillWithUniqueIds) ?? [],
+  };
+}
+
+type Props = {
+  projectSlug: string;
+  viewHierarchies: EventAttachment[];
+};
+
+function EventViewHierarchy({projectSlug, viewHierarchies}: Props) {
+  const [selectedViewHierarchy] = useState(0);
```

Comment:
can we have multiple view hierarchies for a single event? is it the same situation as screenshots?

Reply:
Yeah, in the initial conversations I was told there could be multiple view hierarchies so I added this state variable as a placeholder while this is still early stage.
I'm going to clarify, update my project doc, and follow this up in another PR whether it be removing it or adding some UI to handle changing the variable